integration/src/test/scala/ammonite/integration/StandaloneTests.scala

```diff
@@ -16,28 +16,38 @@ import utest.framework.TestSuite
 object StandaloneTests extends TestSuite{
   // Prepare standalone executable
   val scalaVersion = scala.util.Properties.versionNumberString
+
   val tests = TestSuite {
     val ammVersion = ammonite.Constants.version
     val executableName = s"ammonite-repl-$ammVersion-$scalaVersion"
     val Seq(executable) = ls.rec! cwd |? (_.last == executableName)
-    val resources = cwd/'repl/'src/'test/'resources/'standalone
+    val replStandaloneResources = cwd/'repl/'src/'test/'resources/'standalone
+    val shellAmmoniteResources = cwd/'shell/'src/'main/'resources/'ammonite/'shell
+    //use Symbol to wrap symbols with dashes.
+    val emptyPrefdef = shellAmmoniteResources/Symbol("empty-predef.scala")
```

Comment:
You can just use the string `"empty-predef.scala"` here, no need the `Symbol` wrapper